dashboard/lib/services/lti.rb

```diff
@@ -4,24 +4,34 @@
 
 class Services::Lti
   def self.initialize_lti_user(id_token)
-    custom_claims = id_token[Policies::Lti::LTI_CUSTOM_CLAIMS]
     user_type = Policies::Lti.get_account_type(id_token)
+    teacher_keys = [:name, :display_name, :full_name, :family_name, :given_name]
+    student_keys = [:name, :display_name, :full_name, :given_name]
```

Comment:
Maybe rename these to `teacher_name_keys` and `student_name_keys`? Also, is it necessary to maintain two separate lists of keys, or since the student list is a subset of the teacher list, can we just use the larger list and rely on the fallback logic in `get_claim_from_list`?

Reply:
✅ on the naming suggestion.

For the two separate lists, there isn't a case where we'd use the family_name for a student, as this is an additional value we set for students. We include it as a possibility for teachers, since they might go by their last name (family_name), so the order of preference is family_name before given_name for them.

But, I agree maintaining two lists here is a bit clunky. But it felt more explicit than duplicating the array, cause arrays in Ruby are mutable across variable reassignments unless explicitly cloned.

Reply:
Oh ok, that makes sense. I missed that logic when I was reading through.